deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -66,6 +66,12 @@ spec:
           - get
           - list
           - update
+        - apiGroups:
+            - certificates.k8s.io
```

Comment:
I think this is a bug that you uncovered in existing code path, you need to have a separate commit for this unless you're trying to fix the same thing in this commit?

Reply:
So is a separate commit needed for this? I don't fully understand @ravisantoshgudimetla's comment. What does "same thing" refer to here?